test/functional/wallet_bumpfee.py

```diff
@@ -38,7 +38,6 @@ def set_test_params(self):
             "-walletrbf={}".format(i),
             "-mintxfee=0.00002",
             "-deprecatedrpc=totalFee",
-            "-addresstype=p2sh-segwit", # TODO update constants in test and remove
```

Comment:
I don't understand why we only test this for the default addresstype. I'd prefer to either hardcode the addresstype, so that it doesn't break when the default changes, or to test all relevant address types.

Reply:
Fair point -- from the comment and issue #17043 I was concluding that it was even desired if tests break after the default address changes, to increase the pressure to write/correct tests for it (which could otherwise be forgotten), but then again I have to admit that this is kind of a weak argument. Of course it would be best to have tests for all relevant address types. For now I will change the type to bech32.